src/main/webapp/index.html

```diff
@@ -36,19 +45,36 @@ <h1>
         <label id="show-mutation-text" for="show-mutations">Select this checkbox to show the mutated nodes in the
           graph</label><br>
         <input type="checkbox" id="show-mutations" name="show-mutations" checked=true>
+
+        <!-- Slider code begins -->
+        <label id="slider-text" for="slider">Use the slider to adjust the index of the currently displayed
+          mutation</label>
+        <div id="slider" class="mdc-slider mdc-slider--discrete mdc-slider--display-markers" tabindex="0" role="slider"
+          aria-valuemin="0" aria-valuemax="0" aria-valuenow="0" aria-label="Select Value">
+          <div class="mdc-slider__track-container">
+            <div class="mdc-slider__track"></div>
+            <div class="mdc-slider__track-marker-container"></div>
+          </div>
+          <div class="mdc-slider__thumb-container">
+            <div class="mdc-slider__pin">
+              <span class="mdc-slider__pin-value-marker"></span>
+            </div>
+            <svg class="mdc-slider__thumb" width="21" height="21">
+              <circle cx="10.5" cy="10.5" r="7.875"></circle>
+            </svg>
+            <div class="mdc-slider__focus-ring"></div>
+          </div>
+        </div>
+        <!-- Slider code ends -->
       </div>
-      <label>Search by Node: </label>
-      <input type="text" id="search" name="search"></input>
-      <button id="search-button">Search</button>
-      <label id="search-error"></label>
     </div>
     <!-- COLUMN 2: includes messages -->
     <div class="graph-content-column" id="graph-content-logs">
       <ul id="log-list">
-        <li class="log-msg">Welcome to our application! Here you'll see a log of messages pertaining to the graph on the left. Enjoy! :)</li>
+        <li>Welcome to our application! Here you'll see a log of messages pertaining to the graph on the left. Enjoy! :)
```

Comment:
do you think we would ever make use of the class?

Reply:
Oops I think it got deleted by accident while merging. Sorry!